docs/2-fork-accountability.md

```diff
@@ -0,0 +1,163 @@
+However, we rely on the fact that Tendermint messages are signed and it is not possible to forge messages (i.e., some process *p* cannot state that it has received a message from process *p'* if *p'* did not send that message). 
+
+So, excluding the previous problem, the following situations are possible:
+ 
+- A process *p* denies having received a message *m* (*p*'s message logs does not contain *m* as a message received): this does not really affect the monitor's job because it does not contribute to create a fork and we can ignore this scenario  
+- A process *p* denies having sent a message *m* (*p*'s message logs does not contain *m* as a message sent): this can lead to a fork so we need to consider the fact that processes can do this to hide their faultiness. However, if *m* led to a fork, at least one correct process has received that message and we can identify the faulty process. 
+
+Given this overview, it is now clear why the monitor would just need the message logs from the correct processes: if a message *m* from a process *p'* has been received by a process *p* and *p'* denies sending m, we can add this missing message in *p'* 's message logs without needing to take any other action. 
+
+Therefore, we know that the monitor can "infer" the valid message logs for each process after receiving the message logs from all the correct processes. The next step would be to simply analyze the logs for each process and determine whether it is faulty or not.
+However, we still need to ensure that we are going to receive the message logs from the correct processes and this depends on how we model the communication between the monitor and validators.
+
+### Communication between the monitor and validators
+
+The communication between the monitor and the validators for receiving message logs of the Tendermint consensus algorithm is a crucial aspect of the monitor execution.
+
+For example, what happens when the monitor does not receive message logs from one of the validators? Should this validator be considered faulty?
+
+#### Synchronous model
+
+If we model the communication with a synchronous model, the only option would be waiting some time to receive all the message logs before running the accountability algorithm.
+After that time, if some process p did not send its message logs, the only thing monitor can do is considering p faulty, even though it does not have all the information to determine its faultiness with certainty. 
+On the other hand, if it does not consider p faulty, the monitor might not be able to find all the faulty processes that led to a fork.
+
+#### Asynchronous model
+
+This method would work but we want to make no assumptions regarding the communication between the monitor and the validators. 
+If we do not make any assumption regarding the communication, we would realize that it is impossible to design an accountability algorithm where the communication between monitor and validators is completely asynchronous.
```

Comment:
Let's tone down a rhetoric here with "it is impossible to design an accountability algorithm".
The reason is that this is highly not trivial because we would need to be extremely careful with defining a consensus algorithms that we want to "transform" into accountable algorithms.

So, my suggestion is to say: The approach given above does not work in this case.

Reply:
Fixed